examples/100-formula-parse/example-formulas.md

```diff
@@ -161,7 +169,7 @@ and what types we see after parsing the information:
 [testmark]:# (hello-and-debug/formula.debug)
 ```text
 struct<FormulaAndContext>{
-	formula: struct<Formula>{
+	formula: union<FormulaCapsule>{struct<Formula>{
```

Comment:
It's kinda neat how this debug printout syntax collapses the union/variant into zero additional lines nor indentation.

Kinda wish we could get the JSON output to do that too.  I bet it would make the JSON look a lot prettier.